lib/scenic/adapters/postgres.rb

```diff
@@ -166,6 +171,16 @@ def try_index_create(index)
         connection.execute("ROLLBACK TO SAVEPOINT #{index.index_name}")
         "index '#{index.index_name}' on '#{index.object_name}' is no longer valid and has been dropped."
       end
+
+      def version_number
+        @version_number ||= connection.select_value("select version()")
+          .match(/PostgreSQL (\d.\d.\d)/).captures[0].tr(".", "").to_i
+      end
+
+      def raise_unless_materialized_views_supported
+        msg = "In order to use materialized views you must use PostgreSQL version 9.3 or newer"
+        raise StandardError, msg unless version_number >= 930
```

Comment:
I think we'll still want to wrap this as `if version_number < 930`.

@derekprior do we want custom exceptions? These probably aren't being rescued so maybe not an issue?